src/utils.js

```diff
@@ -73,7 +73,7 @@ export function saferEvalNoReturn(expression, dataContext, additionalHelperVaria
     )
 }
 
-const xAttrRE = /^x-(on|bind|data|text|html|model|if|for|show|cloak|transition|ref)\b/
+const xAttrRE = /^(data-x|x)-(on|bind|data|text|html|model|if|for|show|cloak|transition|ref)\b/
```

Comment:
Not a blocker but I thought it was worth sharing:
we can use a non-capturing block in this regex
`const xAttrRE = /^(?:data-x|x)-(on|bind|data|text|html|model|if|for|show|cloak|transition|ref)\b/`
and keep using `typeMatch[1]` later on

Reply:
Yeah good spot. This regex was the first one I tried so I just kept it as is, might be worth changing though.

Reply:
Ooo lookbehind right? Will it get transpiled (correctly)?

Reply:
It's slightly different. I think lookbehind is not supported everywhere but non-capturing groups are. It matches the group but it doesn't add a record in the resulting array.

Reply:
Ah yes I see there's a one char difference 😄 Regex are always a fun one